doc/source/conf.py

```diff
@@ -311,13 +311,19 @@
         "json_url": f"https://{cname}/versions.json",
         "version_match": switcher_version,
     },
-    "cheatsheet": {
+}
+
+BUILD_CHEATSHEET = (
+    True if os.environ.get("BUILD_CHEATSHEET", "true") == "true" else False
```

Comment:
Is the ``os.environ.get`` case sensitive? If it's the case, we need to modify `true` by `True`.

Reply:
Very good catch. I think it is.

Reply:
```suggestion
    True if os.environ.get("BUILD_CHEATSHEET", "true").lower() == "true" else False
```

Reply:
I guess default should be ``False``, except in CICD. If CICD is failing because of the cheat sheet, we can investigate by setting it to ``True`` locally.